core/errors.ml

```diff
@@ -184,8 +184,8 @@ let format_exception =
      pos_prefix (Printf.sprintf "Error: Cannot load plugin dependency '%s' (link error: %s)\n" file (Dynlink.error_message err))
   | LoadFailure (file, err) ->
      pos_prefix (Printf.sprintf "Error: Cannot load plugin '%s' (link error: %s)\n" file (Dynlink.error_message err))
-  | ClientCallOutsideWebMode fn ->
-     pos_prefix (Printf.sprintf "Error: Cannot call client side function '%s' outside of web mode\n" fn)
+  | ForbiddenClientCall (fn, reason) -> 
```

Comment:
One more tiny thing: could you remove the space at the end of this line?  This is why the CI is failing at the moment.